CMakeSettings.json

```diff
@@ -85,7 +97,7 @@
     {
       "name": "x64-Release",
       "generator": "Ninja",
-      "configurationType": "RelWithDebInfo",
+      "configurationType": "Release",
```

Comment:
Are you sure?
As far as I understand (fixme) our previous releases were built with DebInfo, which now goes away.
At first look this makes sense for me, too, although I think this sacrifices the possibility of getting stack traces in case of production support.

Reply:
Yep, that's why x64-RelWithDebInfo was added.

Reply:
Fair enough :)